ts/screens/wallet/payment/ManualDataInsertionScreen.tsx

```diff
@@ -270,7 +268,9 @@ class ManualDataInsertionScreen extends React.Component<Props, State> {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  navigateToWalletHome: () => dispatch(navigateToWalletHome()),
+  navigateBack: () => {
```

Comment:
```suggestion
  goBack: () => {
    dispatch(navigateBack());
  }
```

import action from `import { navigateBack } from "../../../store/actions/navigation";`

Reply:
Done!